tests/openfhe/end_to_end/README.md

```diff
@@ -0,0 +1,40 @@
+# End to end Rust codegen tests
```

Comment:
nit: i think this was supposed to change to OpenFHE codegen and links to openfhe instead?

Reply:
oh, whoops, the end-to-end part was meant to be in a subsequent PR, not this one. So everything in this directory is just random partial work. Let me clean it up.